dataactvalidator/app.py

```diff
@@ -1,111 +1,59 @@
 import logging
 
-import csv
-from flask import Flask, request, g
+import boto3
+from flask import Flask, g, current_app
 
 from dataactcore.config import CONFIG_BROKER, CONFIG_SERVICES
 from dataactcore.interfaces.db import GlobalDB
-from dataactcore.interfaces.function_bag import write_file_error, mark_job_status
 from dataactcore.logging import configure_logging
-from dataactcore.models.jobModels import Job
-from dataactcore.utils.jsonResponse import JsonResponse
-from dataactcore.utils.responseException import ResponseException
-from dataactvalidator.validation_handlers.validationError import ValidationError
 from dataactvalidator.validation_handlers.validationManager import ValidationManager
 
 
 logger = logging.getLogger(__name__)
+logging.getLogger('boto3').setLevel(logging.CRITICAL)
+logging.getLogger('botocore').setLevel(logging.CRITICAL)
 
 
 def create_app():
-    """Create the Flask app."""
-    flask_app = Flask(__name__.split('.')[0])
-    flask_app.debug = CONFIG_SERVICES['debug']
-    local = CONFIG_BROKER['local']
-    error_report_path = CONFIG_SERVICES['error_report_path']
-    flask_app.config.from_object(__name__)
-
-    # Future: Override config w/ environment variable, if set
-    flask_app.config.from_envvar('VALIDATOR_SETTINGS', silent=True)
-
-    @flask_app.teardown_appcontext
-    def teardown_appcontext(exception):
-        GlobalDB.close()
-
-    @flask_app.before_request
-    def before_request():
-        GlobalDB.db()
-
-    @flask_app.errorhandler(ResponseException)
-    def handle_response_exception(error):
-        """Handle exceptions explicitly raised during validation."""
-        logger.error(str(error))
-
-        job = get_current_job()
-        if job:
-            if job.filename is not None:
-                # insert file-level error info to the database
-                write_file_error(job.job_id, job.filename, error.errorType, error.extraInfo)
-            if error.errorType != ValidationError.jobError:
-                # job pass prerequisites for validation, but an error
-                # happened somewhere. mark job as 'invalid'
-                mark_job_status(job.job_id, 'invalid')
-        return JsonResponse.error(error, error.status)
-
-    @flask_app.errorhandler(Exception)
-    def handle_validation_exception(error):
```

Comment:
One note here -- an exception will go all the way up to the Python run time. You've got supervisor set up, though, so those processes _should_ get restarted.